openverse_catalog/dags/providers/provider_api_scripts/freesound.py

```diff
@@ -0,0 +1,280 @@
+"""
+Content Provider:       Freesound
+
+ETL Process:            Use the API to identify all CC-licensed images.
+
+Output:                 TSV file containing the image, the respective
+                        meta-data.
+
+Notes:                  https://freesound.org/apiv2/search/text'
+                        No rate limit specified.
+"""
+import copy
+import functools
+import logging
+import os
+from datetime import datetime
+
+import requests
+from common.licenses.licenses import get_license_info
+from common.loader import provider_details as prov
+from common.requester import DelayedRequester
+from common.storage.audio import AudioStore
+
+
+LIMIT = 150
+DELAY = 1  # in seconds
+RETRIES = 3
+HOST = "freesound.org"
+ENDPOINT = f"https://{HOST}/apiv2/search/text"
+PROVIDER = prov.FREESOUND_DEFAULT_PROVIDER
+# Freesound only has 'sounds'
+FREESOUND_CATEGORY = "sound"
+API_KEY = os.getenv("FREESOUND_API_KEY", "not_set")
```

Comment:
The key needs to be added to the env.template file.

Reply:
Great catch, @krysal , thank you!